tests/smoke/base_deploy_test.go

```diff
@@ -26,17 +26,20 @@ import (
 var (
 	logger = capnslog.NewPackageLogger("github.com/rook/rook", "integrationTest")
 
-	defaultNamespace     = "default"
-	defaultRookNamespace = "test-rook"
-	clusterNamespace1    = "test-cluster-1"
-	clusterNamespace2    = "test-cluster-2"
+	defaultNamespace           = "default"
+	defaultRookSystemNamespace = "rook-system"
+	defaultRookNamespace       = "rook"
```

Comment:
can we leave this as "test-rook" so that we are testing deploying rook to other namespaces.

Reply:
the issue is that when someone uses the `skip-rook-install` flag, the int-test will use this namespace to get the cluster. I think we use `rook` or we add a way for users to specify the namespace when using `skip-rook-install` flag